src/main/java/trivia/QuestionCategoryName.java

```diff
@@ -0,0 +1,24 @@
+package trivia;
+
+enum QuestionCategoryName {
+
+  POP("Pop"),
+  SCIENCE("Science"),
+  SPORTS("Sports"),
+  ROCK("Rock");
+
+  private final String name;
+
+  QuestionCategoryName(String name) {
+    this.name = name;
+  }
+
+  public QuestionCategory createQuestionList() {
```

Comment:
Enum category is just abstraction around category, it should not be responsible for creating/maintaining question decks. It's actually other way around, Decks are referencing  categories.